packages/core/app-layout/src/components/collapsibleSection/CollapsibleSection.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <component
+    :is="collapsible ? 'details' : 'div'"
+    class="collapsible-section"
+    data-testid="collapsible-section"
+  >
+    <component
+      :is="collapsible ? 'summary' : 'div'"
+      class="collapsible-section-header"
+      data-testid="collapsible-section-header"
+    >
+      <slot name="header">
+        <div class="collapsible-section-default-header">
+          <div
+            v-if="title"
+            class="collapsible-section-title"
+          >
+            {{ title }}
+          </div>
+          <div
+            v-if="description"
+            class="collapsible-section-description"
+          >
+            {{ description }}
+          </div>
+        </div>
+      </slot>
+
+      <ChevronRightIcon
+        v-if="collapsible"
+        class="collapsible-section-chevron-icon"
+        decorative
+      />
+      <slot
+        v-else
+        name="actions"
+      />
+    </component>
+
+    <div
+      v-if="$slots.default"
+      class="collapsible-section-content"
+      data-testid="collapsible-section-content"
+    >
+      <slot />
```

Comment:
nit: just for readability
```suggestion
      <slot name="default" />
```